http/src/main/resources/leo.conf

```diff
@@ -123,7 +123,7 @@ akka {
         },
         {
           # Default trust store
-          path: ${JAVA_HOME}/lib/security/cacerts
+          path: ${?JAVA_HOME}/lib/security/cacerts
```

Comment:
Same with this - is this intentional?

Reply:
ohh yeah, I can get rid of them cuz they're out of the scope of this change, but leo tests don't run locally without JAVA_HOME being defined

Reply:
I'm okay with the java changes going in - it doesn't affect anything other than the local tests.